descwl_coadd/coadd.py

```diff
@@ -64,7 +67,9 @@ class MultiBandCoadds(object):
     interp_bright: bool
         If True, mark BRIGHT as SAT and do interpolation
     replace_bright: bool
-        If True, replace BRIGHT with nois.
+        If True, replace BRIGHT with noise.
+    max_mask_frac : float
```

Comment:
this is not even used yet, not sure why I have it there

Reply:
kk i'll remove